skore/src/skore/sklearn/cross_validation/cross_validation_reporter.py

```diff
@@ -124,6 +124,7 @@ def __init__(self, *args, **kwargs):
         self.estimator = args[0] if len(args) >= 1 else kwargs.get("estimator")
```

Comment:
`estimator seems to be mandatory?

```suggestion
        self.estimator = args[0] if len(args) >= 1 else kwargs["estimator"]
```

Reply:
See comments about X